### DISCUSSION TITLE
Should the doc be updated wrt. `nerdctl compose --profiles` ?

### DISCUSSION THREAD

**Discussion by dimaqq** [containerd/nerdctl#2742] [General]
The doc https://github.com/containerd/nerdctl/blob/main/docs/compose.md seems to suggest that profiles are not implemented in docker composition.

Yet, this PR https://github.com/containerd/nerdctl/pull/2042 was merged some time ago, so maybe profiles are in fact supported?

**Comment by AkihiroSuda** [comment]
Yes, the docs has to be updated

**Comment by dimaqq** [comment]
Thanks I've posted https://github.com/containerd/nerdctl/pull/2750
